prototypes/item.lua

```diff
@@ -4,76 +4,216 @@
  * See License.txt in the project directory for license information.
 --]]
 data:extend({
+	-- Basic Warehouse
 	{
 		type = "item",
 		name = "warehouse-basic",
 		icon = "__Warehousing__/graphics/icons/warehouse-basic.png",
 		icon_size = 32,
 		subgroup = "storage",
-		order = "a[items]-c[warehouse]",
+		order = "a[items]-c[warehouse]-a[base]",
 		place_result = "warehouse-basic",
-		stack_size = 15,
+		stack_size = 15
```

Comment:
Please leave trailing commas alone, and use them in the items/tables you add. 😸

This goes for all files, but I'm only going to make the comment once.

Reply:
This I totally know is an unwritten rule and I just didn't catch it but yes I will try to remember to look for this.

Reply:
Did your IDE remove the commas in some auto-format command? lol